doc/radosgw/uadk-accel.rst

```diff
@@ -0,0 +1,131 @@
+===============================================
+UADK Acceleration for Compression
+===============================================
+
+UADK is a framework for user application to access hardware accelerator in a
+unified, secure, and efficient way. UADK is comprised of UACCE, libwd and many
+other algorithm libraries for different applications.
+
+See `Compressor UADK Support`_.
+
+
+UADK in the Software Stack
+=========================
```

Comment:
Add an =

Reply:
This is already present upon my 04 Dec 2024 review.